ScenePlayer/build.gradle

```diff
@@ -6,12 +6,12 @@ dependencies {
 }
 
 android {
-    compileSdkVersion 21
-    buildToolsVersion "21.1.2"
+    compileSdkVersion 23
+    buildToolsVersion "23.0.1"
 
     defaultConfig {
         minSdkVersion 10
-        targetSdkVersion 21
+        targetSdkVersion 23
```

Comment:
Ok. I'll test it on an api level 23 emulator and if it doesn't work well
I'll change it to 22

On Sun, 8 Nov 2015 20:55 Joe Bowbeer <notifications@github.com> wrote:

> In ScenePlayer/build.gradle
> <https://github.com/libpd/pd-for-android/pull/16#discussion_r44229420>:
>
> >
> >      defaultConfig {
> >          minSdkVersion 10
> > -        targetSdkVersion 21
> > +        targetSdkVersion 23
>
> ditto - make this 22 if permission handling is not updated for 23.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/libpd/pd-for-android/pull/16/files#r44229420>.
>